# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux-header/ps3fb-cursor.patch
# Copyright (C) 2019 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-5.0/arch/powerpc/include/uapi/asm/ps3fb.h.vanilla	2019-07-14 14:54:38.629321697 +0000
+++ linux-5.0/arch/powerpc/include/uapi/asm/ps3fb.h	2019-07-14 15:11:45.477317448 +0000
@@ -34,4 +34,8 @@
 
+#define PS3FB_IOCTL_CURSOR_ENABLE _IOW('r', 10, int)	/* cursor enable/disable */
+#define PS3FB_IOCTL_CURSOR_POS    _IOW('r', 11, int)	/* cursor x/y pos*/
+#define PS3FB_IOCTL_CURSOR_OFFS   _IOW('r', 12, int)	/* cursor data offset */
+
 #ifndef FBIO_WAITFORVSYNC
 #define FBIO_WAITFORVSYNC         _IOW('F', 0x20, __u32) /* wait for vsync */
 #endif
